python/src/worker/tasks/cluster_cells.py

```diff
@@ -41,7 +43,33 @@ def _format_result(self, raw):
                     "cellIds": list(view.map(int)),
                 }
             )
-        return [Result(json.dumps(cell_set), cacheable=False)]
+        
+        return cell_set
+    
+    def _update_through_api(self, cell_set_object):
+        r = requests.patch(
+            f"{config.API_URL}/v1/experiments/{self.request['experimentId']}/cellSets",
+            headers={
+                "content-type": "application/boschni-json-merger+json",
+                "Authorization": self.request["Authorization"],
+            },
+            json=[
+                {
+                    "$match": {
+                        "query": f'$[?(@.key == "{self.task_def["cellSetKey"]}")]',
+                        "$remove": True,
+                    },
+                },
+                {
+                    "$prepend": cell_set_object,
+                }
+            ],
+        )
+
+        info(r.status_code)
+    
+    def _format_result(self, cell_set_object):
+        return [Result(json.dumps(cell_set_object))]
```

Comment:
Set `cacheable` to be `False` for clustering to always run (needed for `update_through_api`).